en/dev_setup/dev_env.md

```diff
@@ -18,6 +18,7 @@ Target | Linux (Ubuntu) | Mac | Windows
 **Simulation:** [Gazebo SITL](../sim_gazebo_gz/README.md) | &check; | &check; | &check;
 **Simulation:** [Gazebo Classic SITL](../sim_gazebo_classic/README.md) | &check; | &check; | &check;
 **Simulation:** [ROS with Gazebo Classic](../simulation/ros_interface.md) | &check; | | 
+**Simulation:** ROS2 with Gazebo | &check; | | 
```

Comment:
PS I added this note so that it is clear we support ROS2 as well. Whether we actually need a topic like the one for ROS1 (linked in point above) that explains the ROS / Simulator interface is something I leave to you to consider.